src/app/profiles/App/Pages/Profiles/Profile/ProfileNoticeListItem.tsx

```diff
@@ -104,7 +104,7 @@ export const ProfileNoticeListItem = ({
           <Trans i18nKey={'profiles:notice.pined_on'}>
             <NoticeHighlight>
               Message épinglé sur
-              <NoticeURL>{exampleMatchingUrl}</NoticeURL>
+              <NoticeURL>{{ exampleMatchingUrl }}</NoticeURL>
```

Comment:
I might miss something here... :thinking: 
From what I see `NoticeURL` is a `span` and `exampleMatchingUrl` is string so one pair of bracket should be enough to displayt this link. 
Aren't these double bracket meaning we're passing an object here?
```
{
    exampleMatchingUrl: 'www.dismoi.io'
}
```

Reply:
I thought the same thing at first, but i18n needs an object and we are in a Trans component